lesson_7/removeElement.js

```diff
@@ -0,0 +1,16 @@
+const array = [1, 2, 3, 4, 5, 6, 7];
+
+/**
+ * 
+ * @param {array} someArray 
```

Comment:
to show array elements type
```suggestion
 * @param {Array<number>} someArray 
```